src/datagrid/DataGrid.stories.tsx

```diff
@@ -67,4 +78,29 @@ storiesOf("DataGrid", module)
                 }}
             />
         </div>
+    ))
+    .add("Grid with sorting", () => (
+        <div style={{width: "80%"}}>
+            <DataGrid columns={sortColumns} data={normalRows} footer={footer} />
+        </div>
+    ))
+    .add("Grid with filter", () => (
+        <div style={{width: "80%"}}>
+            <DataGrid columns={filterColumns} data={normalRows} footer={footer} />
+        </div>
+    ))
+    .add("Grid with sorting and filter", () => (
+        <div style={{width: "80%"}}>
+            <DataGrid columns={sortAndFilterColumns} data={normalRows} footer={footer} />
+        </div>
+    ))
+    .add("Grid with expandable row", () => (
```

Comment:
On adding both multi-select and expandable, expandable icon comes before select option. In invision, select option comes before expandable icon.

Reply:
@payald  Thanks for noticing this. I have fixed it now.